src/test/java/seedu/address/logic/commands/room/FindRoomCommandTest.java

```diff
@@ -25,9 +27,16 @@ void execute_numberOfRooms_notDefined() {
     void execute_numberOfRooms_success() {
         Model model = new ModelManager(new PatientRecords(), new UserPrefs(), new RoomList());
         Model expectedModel = new ModelManager(model.getPatientRecords(), new UserPrefs(), new RoomList());
+        Predicate<Room> predicate = preparePredicate(new Room(1));
+        expectedModel.updateFilteredRoomList(predicate);
         model.addRooms(100);
         expectedModel.addRooms(100);
         String expectedMessage = String.format(FindRoomCommand.MESSAGE_SUCCESS, new Room(1));
         assertCommandSuccess(new FindRoomCommand(), model, expectedMessage, expectedModel);
     }
+
+    private Predicate<Room> preparePredicate(Room room) {
+        Predicate<Room> predicate = room1 -> room1.getRoomNumber() == room.getRoomNumber();
```

Comment:
Perhaps the predicate could take on a more meaningful name. What about `filterByRoomNumberOne` or something along those lines?

Reply:
If we are changing that, I think we can make the method name clearer too. Maybe `getFilterByRoomNumberPredicate()`?